cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/BuildTime.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.cli.utils;
+
+/**
+ * Util class to capture build time information.
+ *
+ * @since 2.0.0
+ */
+public class BuildTime {
+
+    private static final BuildTime instance = new BuildTime();
+
+    public long timestamp;
+    public boolean offline;
+    public long projectLoadDuration;
+    public long packageResolutionDuration;
+    public long packageCompilationDuration;
+    public long codeGenDuration;
+    public long testingExecutionDuration;
+    public long totalDuration;
```

Comment:
Why we are not using getters and setters here? So we can remove `URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD` exclude.

Reply:
This class is internal to the CLI module. We don't export this in the module. So I think there is no harm in having them as public.